packages/react/src/stories/ic-button.stories.mdx

```diff
@@ -1075,3 +1075,22 @@ export const PopoverDropdown = () => {
     )}
   </Story>
 </Canvas>
+
+### File Upload
```

Comment:
```suggestion
### File upload
```

Other stories in button are all sentence case. Please can you change the story name to File upload.